superset-frontend/src/SqlLab/components/SqlEditor/index.tsx

```diff
@@ -0,0 +1,742 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { FC, useState, useEffect, useRef } from 'react';
+import { CSSTransition } from 'react-transition-group';
+import { useDispatch, useSelector } from 'react-redux';
+import Split from 'react-split';
+import { t, styled, useTheme } from '@superset-ui/core';
+import debounce from 'lodash/debounce';
+import throttle from 'lodash/throttle';
+import StyledModal from 'src/components/Modal';
+import Mousetrap from 'mousetrap';
+import Button from 'src/components/Button';
+import Timer from 'src/components/Timer';
+import {
+  Dropdown,
+  Menu as AntdMenu,
+  Menu,
+  Switch,
+  Input,
+} from 'src/common/components';
```

Comment:
The components `Dropdown` and `Switch` are available in the `src/components` directory. I think we should prefer those to keep consistency.

Reply:
![image](https://user-images.githubusercontent.com/29536522/150847477-9e5752fd-61b5-4b32-93c8-a4d94116bccd.png)
@geido Dropdown UI is changed

Reply:
I think I should revert these changes